man/man8/zpool-reopen.8

```diff
@@ -30,26 +30,24 @@
 .Dd August 9, 2019
 .Dt ZPOOL-REOPEN 8
 .Os
+.
 .Sh NAME
 .Nm zpool-reopen
-.Nd Reopen all virtual devices (vdevs) associated with a ZFS storage pool
+.Nd reopen vdevs associated with ZFS storage pool
```

Comment:
nit: I'd keep the "a" here at least.

Reply:
title => article (especially "a") is left out

Reply:
I don't think these are titles in that sense. The "BSD System Manager's Manual" bit is a title, but "reopen vdevs ..." is not. It's not in title case. It's more of a sentence fragment. In any event, I think it both flows better and subtly conveys useful information to keep the relevant pluralizations here.

Reply:
Oh, it seems that zpool-reopen actually supports multiple (and all) pools, which the manpage didn't mention. You're right about the pluralisation – that's what I was doing, though, clearly, not well – I re-did the titles, where applicable, to pluralise the right parts right.

If you wanna get super pedantic about it, then yeah, "BSD System Manager's Manual" is the title of the book. The "page" itself has a title(? it's 4am, I'm not looking at RUNOFF source) heading (`.TH`), too, since it spans multiple pages of the book (hence, you know, being in the corners with "BSD S M M"), and the name sexion is a user-visible title (or, you know, main heading), too, in the `name - desc` format. Three titles, three casings, one grammatical convention (I'm discounting "BIG-NAME" for "grammar" here). Strictly adhering to title grammar is shaky at best, but it's well-enforced in linux man-pages, for example, and the few pages from FreeBSD/P9/util-linux I looked at do a relatively good job as well.

Reply:
I still don't agree, but I'm not going to fight about it any more.